internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Pull Request Review apps
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the app.
+
+If a pull request is created from a fork of this repo (typical if the
```

Comment:
```suggestion
If you create a pull request from a fork of this repo (typical if the
```

Reply:
Suggesting we reword in second person as that can sometimes help personalise content for users.